README.md

```diff
@@ -21,6 +21,9 @@ This module has been tested to work on the following systems with Puppet v3.x on
  * Solaris 10
  * Ubuntu 10.04 LTS (Lucid Lynx)
  * Ubuntu 12.04 LTS (Precise Pangolin)
+ * OpenBSD 5.4
```

Comment:
I didn't want to profess to supporting OpenBSD versions which I haven't personally tested Puppet or this module on, but I can add them if you want (although I find it rather unlikely anyone would be using OpenBSD that old, anything older than 2 releases isn't supported).